### PR TITLE
Fix `lxc` in the snap and retry SSH install

### PR DESCRIPTION
**Pull request by Saviq**
Retry SSH installation in case network isn't ready.

### CODE REVIEW

**Comment by Saviq** [comment]
I'm building it here:

https://launchpad.net/~mir-team/+snap/spread-mir-ci

And publishing to the store as `spread-mir-ci`

File: spread/lxd.go [hunk 1]
```diff
@@ -589,6 +600,7 @@ func sshReloadCommand(distro Distro) []string {
 func (p *lxdProvider) tuneSSH(name string, distro Distro) error {
 	cmds := [][]string{
 		{"sed", "-i", `s/^\s*#\?\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/`, "/etc/ssh/sshd_config"},
+		{"/bin/sh", "-c", "[ -d /etc/ssh/sshd_config.d ] && echo PermitRootLogin yes\\\\nPasswordAuthentication yes > /etc/ssh/sshd_config.d/10-spread.conf"},
```
Comment (RAOF) [inline spread/lxd.go:603]: This doesn't get overridden by the later `60-cloudinit-thingy.conf`?

Comment (RAOF) [reply spread/lxd.go:603]: I guess it must not, because WLCS CI is happy.

**Review by RAOF — APPROVED** [review]
Looks sensible

File: spread/lxd.go (hunk 1 above)
Comment (Saviq) [reply spread/lxd.go:603]: Yeah, it's surprising, but it's the first one that's respected.

**Pull request merged**